app/app.go

```diff
@@ -25,12 +26,22 @@ var (
 	tpool *pool.Pool
```

Comment:
Left these private package-level variables here because they should not be instantiable as part of App. They should only exist in one place in the program.